TYPO3.Flow/Migrations/Code/Version20151113161300.php

```diff
@@ -0,0 +1,161 @@
+<?php
+namespace TYPO3\Flow\Core\Migrations;
+
+/*
+ * This file is part of the TYPO3.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use TYPO3\Flow\Configuration\ConfigurationManager;
+use TYPO3\Flow\Security\RequestPattern\ControllerObjectName;
+use TYPO3\Flow\Security\RequestPattern\CsrfProtection;
+use TYPO3\Flow\Security\RequestPattern\Host;
+use TYPO3\Flow\Security\RequestPattern\Ip;
+use TYPO3\Flow\Security\RequestPattern\Uri;
+
+/**
+ * Adjust "Settings.yaml" to new "requestPattern" and "firewall" syntax (see FLOW-412)
+ */
+class Version20151113161300 extends AbstractMigration
```

Comment:
Hey Bastian,

this feels rather strange for me; do I get it right that this is only done by the migration?

I'd rather have pattern identifiers like: "My.Package:Uri" or "My.Package:Ip" -- so it is not confused with Class names.

All the best,
Sebastian